src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>En tom fil är inte tillåten.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Domänen kunde inte hittas.</target>
```

Comment:
So, 3 Sweeds got 3 different translations... "Hosten", "Värddatorn" and "Domänen". 

Ping @pmodin, @magnusnordlander, @morticue, @jongotlin . What is the best translation for "The host could not be resolved." in a error message to the end user? It's the validator for the URL field.